tests/ctl/nmstatectl_test.py

```diff
@@ -57,20 +69,38 @@ def test_run_ctl_directly_show_empty():
     nmstatectl.main()
 
 
-@mock.patch('sys.argv', ['nmstatectl', 'show', 'eth1'])
+@mock.patch('sys.argv', ['nmstatectl', 'show', 'non_existing'])
 @mock.patch.object(
     nmstatectl.netinfo, 'show', lambda: json.loads(LO_JSON_STATE))
 @mock.patch('nmstatectl.nmstatectl.sys.stdout', new_callable=six.StringIO)
 def test_run_ctl_directly_show_only_empty(mock_stdout):
     nmstatectl.main()
-    assert mock_stdout.getvalue() == EMPTY_JSON_STATE
+    assert mock_stdout.getvalue() == EMPTY_YAML_STATE
 
 
 @mock.patch('sys.argv', ['nmstatectl', 'show', 'lo'])
 @mock.patch.object(
     nmstatectl.netinfo, 'show', lambda: json.loads(LO_JSON_STATE))
 @mock.patch('nmstatectl.nmstatectl.sys.stdout', new_callable=six.StringIO)
 def test_run_ctl_directly_show_only(mock_stdout):
+    nmstatectl.main()
+    assert mock_stdout.getvalue() == LO_YAML_STATE
+
+
+@mock.patch('sys.argv', ['nmstatectl', 'show', '--json', 'non_existing'])
```

Comment:
`non_existing_iface`?

Reply:
changed to `non_existing_interface` to match integration tests